### DISCUSSION TITLE
Активность проекта

### DISCUSSION THREAD

**Discussion by leon-win** [vuejs-translations/docs-ru#261] [General]
Всем привет!

Я хочу принять участие в переводе документации Vue.js на русский язык.

Только стал непонятен статус проекта -- очень много PR'ов открытых, такое ощущение, что проводить ревью совсем некому (нет времени или возможности).

Кто-то может рассказать в каком статусе проект?

**Comment by Ibochkarev** [comment]
@leon-win Привет. 

Все ожило и движется в нужном направлении. Самое время присоединиться.
